### PR TITLE
Test cases 

### PR DESCRIPTION
**Pull request by habicll**
Test cases in main

### CODE REVIEW

File: Green City/.godot/scene_groups_cache.cfg
Comment (Bistrel2002) [inline Green City/.godot/scene_groups_cache.cfg:1]: Check this brother

File: Green City/.godot/global_script_class_cache.cfg
Comment (Bistrel2002) [inline Green City/.godot/global_script_class_cache.cfg:1]: What is this?????

**Review by Bistrel2002 — COMMENTED** [review]
do not merge my management document brother

**Review by JasonGROSSO — APPROVED** [review]

**Review by Bistrel2002 — APPROVED** [review]
This is good brother

**Pull request merged**